blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/http/HttpIntegrationTestBase.java

```diff
@@ -259,7 +330,7 @@ public URIBuilder getMetricsURIBuilder() throws URISyntaxException {
 
     public URIBuilder getMetricDataQueryURIBuilder() throws URISyntaxException {
         return new URIBuilder().setScheme("http").setHost("127.0.0.1")
-                .setPort(httpPortQuery).setPath("/v2.0/tenantId/metrics/search")
+                .setPort(httpPortQuery).setPath(getSearchPath)
```

Comment:
@VinnyQ  Won't this leave a `%s` in the path? Should that be replaced with an actual tenant id?